src/lib/s3Uploader.js

```diff
@@ -0,0 +1,45 @@
+import { S3 } from 'aws-sdk';
+
+export const s3 = new S3({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  endpoint: process.env.S3_ENDPOINT,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
```

Comment:
I have another requested change for this as well. In deployed environments, we should pull these variables (as well as the bucket name) out of the `process.env.VCAP_SERVICES` environment variable.